src/common/Vote/styled.js

```diff
@@ -2,41 +2,80 @@ import styled from "styled-components";
 
 export const VoteWrapper = styled.div`
     display: grid;
-    grid-template-columns: auto auto auto 1fr;
-    justify-content: start;
-    align-items: baseline;
+    grid-template-areas: ${({ type }) => {
+        if (type === "header") {
+            return `"icon score range" "count count count"`;
+        }
+        if (type === "details") {
+            return `"icon score range count"`;
+        }
+        return `"icon score count"`;
+    }};
+    flex-wrap: nowrap;
+    column-gap: 12px;
+    row-gap: 17px;
+    justify-content: flex-start;
+    align-items: ${({ type }) => type === "details" ? "baseline" : "center"};
     margin: 0;
+    @media (max-width: ${({ theme }) => theme.breakpoints.mobileLandscape}) {
+        column-gap: 8px;
+        row-gap: 17px;
+        grid-template-areas: "icon score count";
+    }
 `;
 
-export const StarIcon = styled.img`
-    width: 24px;
-    margin-right: 8px;
+export const Icon = styled.img`
+    grid-area: icon;
+    height: ${({ type }) => type === "header" ? "40px" : "24px"};
+    @media (max-width: ${({ theme }) => theme.breakpoints.mobileLandscape}) {
+        height: 16px;
+    }
 `;
 
-export const Score = styled.p`
-    font-size: 22px;
-    color: ${({ theme }) => theme.colors.linkHeader};
+export const Score = styled.span`
+    grid-area: score;
+    display: inline-block;
+    font-size: ${({ type }) => {
+        if(type === "header") {return "30px"}
+        if(type === "details") {return "22px"}
+        return "16px";
+    }};
+    color: ${({ theme, type }) => type === "header" ? theme.colors.voteHeaderScore : theme.colors.voteListScore};
     font-weight: 600;
-    margin: 0 8px 0 0;
+    line-height: 1.5;
+    @media (max-width: ${({ theme }) => theme.breakpoints.tabletLandscape}) {
+        font-size: ${({ type }) => {
+            if(type === "header") {return "14px"}
+            return "13px";
+        }};
+    }
 `;
 
-export const RatingScale = styled.p`
-    font-size: 16px;
-    font-weight: 600;
-    margin: 0;
-    color: ${({ theme }) => theme.colors.linkHeader};
-`;
-
-export const Count = styled.p`
-    padding-left: 4px;
-    font-size: 14px;
-    color: ${({ theme }) => theme.colors.linkHeader};
-    margin: 0 0 0 4px;
+export const Range = styled.span`
+    display: ${({ type }) => type === "header" || type === "details" ? "inline-block" : "none"};
```

Comment:
This solution is also good. And its much easier implement different type of values. Great job.